src/classes/ERR_Notifier.cls

```diff
@@ -38,10 +38,20 @@ public class ERR_Notifier {
 
     /** @description Filtering options to reduce the number of records processed during the notify job */
     @TestVisible
-    private static Integer MAX_RECORDS = 5000;
+    private static Integer MAX_RECORDS = 500;
     @TestVisible
     private static Datetime MAX_AGE_FOR_ERRORS = System.now().addHours(-48);
 
+    /** @description Maximum Heap Storage that will be used to build the message/post body */
+    private Decimal MAX_HEAP_LIMIT {
```

Comment:
Can we add tests to the scenarios where we get near the heap limit? We could set this to 0 in a test and assert that no chatter posts and no emails are sent

Reply:
Good idea. I was thinking about how to test that earlier, but didn't think of just overriding the max.

Reply:
@seanpat09 Unit tests have been added.